### DISCUSSION TITLE
Automatic Incident response within Security Onion 2.3.100

### DISCUSSION THREAD

**Discussion by HermannAndre48** [Security-Onion-Solutions/securityonion#7632] [Unsupported Versions] [answered]
Hello everyone,

I hope you are okay.

I want to know how do you do automatic incident response within Security Onion 2.3.100 ?

Thank you in advance for your answers

**Comment by tomjones1977** [comment]
Incident response is an entire industry in itself.  When you say automatic incident response what do you mean?

**Reply by HermannAndre48** [reply]
Thank you for responding.

Actually, in older versions of Security Onion, there was theHive for automatic incident response. But as of this version, theHive has been replaced by Cases and I would like to know how to do this with Cases. I don't see anything on the internet that shows me how to do this.
This is for a project due next week.

**Comment by dougburks** [comment, marked as answer]
> in older versions of Security Onion, there was theHive for automatic incident response

I'm not sure that I would classify TheHive as `automatic incident response`.

> This is for a project due next week.

If this is for a school project, then I suspect your instructor would want you to work on this yourself.

**Reply by HermannAndre48** [reply]
Thanks you for responding.

The school asked us to use open source software for the project.

I saw on the net that thehive allows automatic response to incidents.

It says in Security Onion version 2.3.100 that thehive is replaced by Cases.

But I don't see on the net how to use Cases for incident response.

**Reply by HermannAndre48** [reply]
Please everyone, i need your help

**Reply by Acewiza** [reply]
"Automatic Response" ≠ "ET DROP Dshield Block Listed Source group 1"

https://docs.securityonion.net/en/2.3/wazuh.html?highlight=response#active-response

DISCLAIMER:  I typed the quote marks, but have no idea who said those things.

**Reply by HermannAndre48** [reply]
Thanks for your responding.

I read wazuh documentation and watch many videos about it but i still don't know how to use wazuh for incident response with Security Onion 2.3.100 ?

Nobody knows ?

**Comment by tomjones1977** [comment]
What exactly are you trying to achieve?  As I said before "automated
incident response" covers far too much to be a useful request.

Wazuh can identify some malicious activity (for example SSH brute force)
and can block it temporarily based on the source IP.  However this is far
from automated incident response.

Take a look at sites such as
https://www.ncsc.gov.uk/collection/incident-management/cyber-incident-response-processes

When you have an idea of what incident response playbook you wish to employ
you can then look at automating specific elements of it.
